ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/map-settings/presentation.tsx

```diff
@@ -30,22 +38,45 @@ const Root = styled.div`
 `
 
 const render = (props: Props) => {
-  const { selected, update } = props
+  const {
+    coordFormat,
+    updateCoordFormat,
+    panOnSearch,
+    updatePanOnSearch,
+  } = props
   return (
     <Root>
-      <Enum
-        options={[
-          { label: 'Degrees, Minutes, Seconds', value: 'degrees' },
-          { label: 'Decimal', value: 'decimal' },
-          { label: 'MGRS', value: 'mgrs' },
-          { label: 'UTM/UPS', value: 'utm' },
-        ]}
-        value={selected}
-        label="Coordinate Format"
-        onChange={update}
-      />
+      <FormGroup row>
+        <FormControlLabel
+          control={
+            <Checkbox
+              checked={panOnSearch}
+              onChange={updatePanOnSearch}
+              color="primary"
+              name="panOnSearch"
+            />
+          }
+          label={<Typography variant="body2">Auto-Pan</Typography>}
+          labelPlacement="start"
+          style={{ paddingLeft: '10px' }}
+        />
+      </FormGroup>
 
-      <ExampleCoordinates {...props} />
+      <div style={{ marginLeft: '-14px' }}>
```

Comment:
📓 The Marionette Region Container in Enum is resistant to styling. This allows me to line it up nicely with the other components.

Reply:
I would just move it to MUI

Reply:
It looks like the whole thing is in an ItemView. When I add a MUI Select instead, it gets hidden behind the settings. I've tried zIndex and a few other things, but I haven't solved it yet.

Here's an example (added a few more options to show the issue):
![image](https://user-images.githubusercontent.com/22667297/107298167-11888200-6a32-11eb-8793-a1f3b482b7ca.png)